restricted_site_access.php

```diff
@@ -1428,6 +1440,49 @@ public static function get_client_ip_address() {
 
 		return $ip;
 	}
+
```

Comment:
we can remove this if we use `wp_kses_post` above, right?

Reply:
I did remove this however let's quickly talk about if is should come back or not :)